apps/accounts/views.py

```diff
@@ -161,7 +161,7 @@ def get(self, request, user, identity, next_path):
             return HttpResponseRedirect(path)
         else:
             login_user(request, user, identity)
-            next_path = next_path or '/'
+            next_path = next_path or reverse('home')
             log.info('Redirecting after login to: {}'.format(next_path))
             return HttpResponseRedirect(next_path)
```

Comment:
`next_path` gets set to `None` if it isn't safe according to django's [`is_safe_url()`](https://github.com/django/django/blob/1.6.11/django/utils/http.py#L250) helper. This happens in the [`@with_user`](https://github.com/mstriemer/olympia/blob/3f7b9ad3989d5c30d4435dc4ea2c87d77905c34b/apps/accounts/views.py#L120-L121) decorator.